locales-he-IL.xml

```diff
@@ -23,29 +23,29 @@
     <term name="and">ו</term>
     <term name="and others">ואחרים</term>
     <term name="anonymous">אלמוני</term>
-    <term name="anonymous" form="short">anon</term>
+    <term name="anonymous" form="short"></term>
```

Comment:
Needs a translation.

Reply:
I have asked a Israeli colleague about abbreviations for bibliographic terms in general ("editor", in specific). The answer was that you normally don't use abbreviations for these terms. Instead, you'd just use the regular term. So here, it would be     
```xml
<term name="anonymous" form="short">אלמוני</term>
```
Perhaps we could do that for all Hebrew terms with missing localization. That's already much better than having English terms here, and if there are complaints we can still correct the terms then.